src/components/settings/UISettings.svelte

```diff
@@ -59,7 +59,7 @@
   {/if}
 </div>
 <CheckOption name="Show timestamps" store={showTimestamp} />
-{#if !isStandalone}
+{#if $displayMode !== DisplayMode.POPOUT}
   <CheckOption name="Show captions" store={showCaption} />
 {/if}
 {#if $showCaption}
```

Comment:
Extra caption settings will always be shown if captions are enabled regardless of display mode.